test/integration/kubernetes_deploy_test.rb

```diff
@@ -1080,6 +1081,24 @@ def test_not_apply_resource_can_be_pruned
     ])
   end
 
+  def test_network_policies_are_deployed_first
+    assert_deploy_success(deploy_fixtures("network-policy"))
+    assert_logs_match_all([
+      "Predeploying priority resources",
+      "Deploying NetworkPolicy/allow-all-network-policy (timeout: 30s)",
+      "Successfully deployed 1 resource",
+      "Successful resources",
+      "NetworkPolicy/allow-all-network-policy",
+    ], in_order: true)
+
+    netpols = networking_v1_kubeclient.get_network_policies(namespace: @namespace)
```

Comment:
Instead of using kubeclient here, please augment the HelloCloud FixtureSet as described in our Readme:

> Add the a basic example of the type to the hello-cloud [fixture set](https://github.com/Shopify/kubernetes-deploy/tree/master/test/fixtures/hello-cloud) and appropriate assertions to `#assert_all_up` in [`hello_cloud.rb`](https://github.com/Shopify/kubernetes-deploy/blob/master/test/helpers/fixture_sets/hello_cloud.rb). This will get you coverage in several existing tests, such as `test_full_hello_cloud_set_deploy_succeeds`.

Reply:
Oh right missed that, sorry. Updated now